pkg/cosign/git/github/github.go

```diff
@@ -82,15 +82,15 @@ func (g *Gh) PutSecret(ctx context.Context, ref string, pf cosign.PassFunc) erro
 
 	passwordSecretEnvResp, err := client.Actions.CreateOrUpdateRepoSecret(ctx, owner, repo, passwordSecretEnv)
 	if err != nil {
-		return errors.Wrap(err, "could not create \"COSIGN_PASSWORD\" environment variable")
+		return errors.Wrap(err, "could not create \"COSIGN_PASSWORD\" github actions secret")
 	}
 
 	if passwordSecretEnvResp.StatusCode < 200 && passwordSecretEnvResp.StatusCode >= 300 {
 		bodyBytes, _ := io.ReadAll(passwordSecretEnvResp.Body)
 		return fmt.Errorf("%s", bodyBytes)
 	}
 
-	fmt.Fprintln(os.Stderr, "Private key written to COSIGN_PASSWORD environment variable")
+	fmt.Fprintln(os.Stdout, "Password written to COSIGN_PASSWORD github actions secret")
```

Comment:
Seems like the rest of the passphrase commands print to stderr, any reason for the switch here?

Reply:
Because what we are printing are not errors. Now you mention it, I'm thinking what would be the reason these are printed to Stderr?

Reply:
Typically logging statements go to stderr, while things intended to be machine-readable or passed to other commands via pipes go to stdout. I don't think there's a clear answer for this command, I just think we should keep it consistent.

Reply:
Agree, let me revert that change for now.